src/components/elements/table/header.js

```diff
@@ -1,21 +1,53 @@
+import { Component } from 'react'
 import cx from 'classnames'
 
-export default ({ api, property }) => {
-  if (!property.columns) {
-    return <thead/>
+export default class Header extends Component {
+  constructor(props) {
```

Comment:
I believe we can use [class properties](https://babeljs.io/docs/plugins/transform-class-properties) instead of setting `state` in the constructor (that way we don't need to add a constructor and remember to call super, etc), like so:

```
export default class Header extends Component {
  state = {
    sortedColumn: null,
    direction: null,
  }
}
```